src/docs/user_toolbar.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Using the dynamic Toolbar in Theia
+---
+
+# Using the dynamic Toolbar in Theia
+
+Eclipse Theia provides a dynamic toolbar providing easy access to commonly used commands. The toolbar contains default commands and can be dynamically adapted by users based on their personal preference.
```

Comment:
I would maybe not use `provides` twice (provides and providing).

Reply:
Can we say "allowing"?